provisionerd/runner/runner.go

```diff
@@ -1057,31 +1030,30 @@ func (r *Runner) runWorkspaceBuild(ctx context.Context) (*proto.CompletedJob, *p
 			},
 		}
 	}
-	if len(planComplete.AiTasks) > 1 {
-		return nil, r.failedWorkspaceBuildf("only one 'coder_ai_task' resource can be provisioned per template")
+
+	if planComplete.AiTaskCount > 1 {
+		return nil, r.failedWorkspaceBuildf("only one 'coder_ai_task' resource can be provisioned per template, found %d", planComplete.AiTaskCount)
 	}
 
-	r.logger.Info(context.Background(), "plan request successful",
-		slog.F("resource_count", len(planComplete.Resources)),
-		slog.F("resources", resourceNames(planComplete.Resources)),
-	)
+	r.logger.Info(context.Background(), "plan request successful")
 	r.flushQueuedLogs(ctx)
 	if commitQuota {
-		failed = r.commitQuota(ctx, planComplete.Resources)
+		failed = r.commitQuota(ctx, planComplete.GetDailyCost())
 		r.flushQueuedLogs(ctx)
 		if failed != nil {
 			return nil, failed
 		}
 	}
 
+	// Run Terraform Apply
 	r.queueLog(ctx, &proto.Log{
 		Source:    proto.LogSource_PROVISIONER_DAEMON,
 		Level:     sdkproto.LogLevel_INFO,
 		Stage:     applyStage,
 		CreatedAt: time.Now().UnixMilli(),
 	})
 
-	resp, failed = r.buildWorkspace(ctx, applyStage, &sdkproto.Request{
+	resp, failed := r.buildWorkspace(ctx, applyStage, &sdkproto.Request{
```

Comment:
buildWorkspace used to abstract plan|apply but now there is a dedicated plan(), so buildWorkspace is just "apply" by a diffent name.